cmd/config-versions.go

```diff
@@ -837,3 +838,6 @@ type serverConfigV31 struct {
 		// Add new external policy enforcements here.
 	} `json:"policy"`
 }
+
+// serverConfigV32 is just like version '31' with added nsq notifer.
+type serverConfigV32 serverConfigV31
```

Comment:
I think this is not necessary, we should explicitly make a new struct for `notifier` i.e rename the existing one to `notifierV3` and then keep the new one as `notifier` with added NSQ. 

Then we can use that in serverConfigV32 and keep the `serverConfigV31` to have `notifierV3` as the Notify struct. 

The reason to do this is to avoid confusion in future.